src/atomate2/forcefields/schemas.py

```diff
@@ -0,0 +1,167 @@
+"""Job to prerelax a structure using an MD Potential."""
+
+import logging
+from typing import List
+
+from pydantic import BaseModel, Field
+
+# from jobflow import Maker, job
+from pymatgen.core.structure import Structure
+
+
+class InputDoc(BaseModel):
+    """The inputs used to run this job."""
+
+    structure: Structure = Field(None, description="The inputted structure.")
+    relax_cell: bool = Field(
+        None,
+        description="Whether cell lattice was allowed to change during relaxation.",
+    )
+    relax_kwargs: dict = Field(
+        None,
+        description="Keyword arguments that passed to :obj:`StructOptimizer.relax`.",
+    )
+    optimizer_kwargs: dict = Field(
+        None,
+        description="Keyword arguments that passed to :obj:`StructOptimizer`.",
+    )
+
+
+class OutputDoc(BaseModel):
+    """The outputs of this job."""
+
+    structure: Structure = Field(None, description="The final, relaxed structure.")
+
+    energy: float = Field(None, description="Total energy in units of eV.")
+
+    energy_per_atom: float = Field(
+        None,
+        description="Energy per atom of the final structure in units of eV/atom.",
+    )
+
+    forces: List[List[float]] = Field(
+        None,
+        description="The force on each atom in units of eV/A for the final structure.",
+    )
+
+    # NOTE: units for stresses were converted to kbar
+    # (* -10 from standard output) to comply with MP convention
+    stress: List[float] = Field(
+        None, description="The stress on the cell in units of kbar (in Voigt notation)."
+    )
+
+    trajectory: dict = Field(
+        None, description="Step-by-step trajectory of the structural relaxation."
+    )
+
+    steps: int = Field(
+        None, description="total number of steps needed to relax the structure."
+    )
+
+
+class FFStructureRelaxDocument(BaseModel):
+    """Document containing information on structure relaxation using a force field."""
+
+    structure: Structure = Field(
+        None, description="Final output structure from the task"
+    )
+
+    input: InputDoc = Field(
+        None, description="The inputted information used to run this job."
+    )
+
+    output: OutputDoc = Field(
+        None, description="The outputted information from this relaxation job."
+    )
+
+    MD_potential: str = Field(
```

Comment:
Lower case attributes!
```suggestion
    md_potential: str = Field(
```